PyInstaller/compat.py

```diff
@@ -863,28 +860,5 @@ def check_requirements():
     Fail hard if any requirement is not met.
     """
     # Fail hard if Python does not have minimum required version
-    if sys.version_info < (3, 3) and sys.version_info[:2] != (2, 7):
-        raise SystemExit('PyInstaller requires at least Python 2.7 or 3.3+.')
-
-
-if not is_py34:
-    class suppress(object):
```

Comment:
This PR cannot be merged int it's current state because it removes this compatibility function. This should not be removed because it is required to run on Python 2.7.

Reply:
Yes, that was a mistake in the rebase that needs sorting.